### DISCUSSION TITLE
logo.png加载速度拉跨，能否改成svg或者允许样式自定义

### DISCUSSION THREAD

**Discussion by xz1127** [AlistGo/alist#5199] [General]
版本3.26.0
请求地址：https://jsd.nn.ci/gh/alist-org/logo@main/logo.png
问题截图[点击查看截图](http://120.25.152.63/alist-logo.png)

**Comment by StartMoney** [comment]
把设置里的logo地址换成其他的就可以了。
比如：https://ghproxy.com/https://raw.githubusercontent.com/alist-org/logo/main/logo.svg

**Reply by xz1127** [reply]
烦请详细说明设置路径。
另外，如截图[点击查看截图](http://120.25.152.63/alist-logo-3.png)三处被删除的“远程地址”对应请求所示，已经做了相关修改（管理/设置/样式/图标+管理/设置/样式/网站图标+管理/设置/样式/首页图表）[点击查看截图](http://120.25.152.63/alist-logo-4.png)，但仍无效果( [点击查看截图](http://120.25.152.63/alist-logo-5.png) )。
从F12看，请求logo.png在文件manifest.json中（[点击查看截图](http://120.25.152.63/alist-logo-2.png)）

**Comment by li-peifeng** [comment]
自己设置logo就行了

**Reply by xz1127** [reply]
试过吗？确定设置了就不会请求那个代码内置的地址吗？

**Reply by li-peifeng** [reply]
如果不放心可以自己编译前端，彻底去除，在/alist-web/src/pages/home/header/Header.tsx

**Reply by xz1127** [reply]
没有不放心，只是加载慢会影响脚本对于页面加载结果的检测

**Reply by li-peifeng** [reply]
> 没有不放心，只是加载慢会影响脚本对于页面加载结果的检测

设置自定义后就不会请求原logo了

**Reply by xz1127** [reply]
没有生效，参考前文https://github.com/alist-org/alist/discussions/5199#discussioncomment-6964627

**Reply by li-peifeng** [reply]
> 没有生效，参考前文https://github.com/alist-org/alist/discussions/5199#discussioncomment-6964627

不知道你是怎么设置的，反正我是设置后f12里是没有原logo的请求的

**Reply by li-peifeng** [reply]
> 没有生效，参考前文https://github.com/alist-org/alist/discussions/5199#discussioncomment-6964627

如果你一定要去掉比如manifest里的地址，唯一的办法就是自己编译
